platform/ui/src/components/AllInOneMenu/__stories__/allInOneMenu.stories.mdx

```diff
@@ -0,0 +1,137 @@
+import AllInOneMenu from '../AllInOneMenu';
+import AllInOneMenuItemPanel from '../AllInOneMenuItemPanel';
+import AllInOneMenuSubMenu from '../AllInOneMenuSubMenu';
+import AllInOneMenuItem from '../AllInOneMenuItem';
+import AllInOneMenuDividerItem from '../AllInOneMenuDividerItem';
+import AllInOneMenuHeaderItem from '../AllInOneMenuHeaderItem';
+import InputRange from '../../InputRange/index.js';
+import SwitchButton from '../../SwitchButton/index.js';
+
+import { ArgsTable, Story, Canvas, Meta } from '@storybook/addon-docs';
+import { createComponentTemplate } from '../../../storybook/functions/create-component-story';
+
+export const argTypes = {
+  component: AllInOneMenu,
+  title: 'Components/AllInOneMenu',
+};
+
+<Meta
+  title="Components/AllInOneMenu"
+  component={AllInOneMenu}
+/>
+
+export const AllInOneMenuTemplate = args => (
+  <div className="w-80">
+    <AllInOneMenu {...args}>
+      <AllInOneMenuItemPanel label="Panel 1">
```

Comment:
where is this label getting used? i don't see that in the UI

Reply:
The label is only needed if there is more than one panel and in that case it is used in the PanelSelector as the label for the tab in the selector. But in this case there is only one panel, so I will make it an optional parameter so someone does not need to use it if there is only one panel.